server/utils/utils.ts

```diff
@@ -149,6 +149,22 @@ export function findTeamMatch(teams: Team[], name: string) {
   )
 }
 
+export function getComponentsForTeam(team: Team) {
+  const components: { componentName: string }[] = []
+  if (team.products && team.products.length > 0) {
+    team.products.forEach((product: Product) => {
```

Comment:
Can probably remove the `if` and just do `team.products?.....` and the same for the `product.components` `if` statement